website/versioned_docs/version-25.x/WatchPlugins.md

```diff
@@ -221,8 +221,16 @@ Any key not used by built-in functionality can be claimed, as you would expect.
 
 Should your plugin attempt to overwrite a reserved key, Jest will error out with a descriptive message, something like:
 
-> Watch plugin YourFaultyPlugin attempted to register key `q`, that is reserved internally for quitting watch mode. Please change the configuration key for this plugin.
+```bash
```

Comment:
This look good 👍 But two notes are missing in all versioned docs. Check `docs/WatchPlugins.md` you added two notes and two code blocks.

Reply:
What's up